services/filesstore/s3store.go

```diff
@@ -47,7 +47,10 @@ const (
 func (b *S3FileBackend) s3New() (*s3.Client, error) {
 	var creds *credentials.Credentials
 
-	if b.accessKey == "" && b.secretKey == "" {
+	isCloud := os.Getenv("MM_CLOUD_INSTALLATION_ID") != ""
```

Comment:
@agnivade, we may want to have a custom env var for this behavior. I could see us continuing to have non-bifrost S3 filestores in the cloud in the future. A unique env var would be ideal for enabling or disabling bifrost in these cases.

Thanks!

Reply:
Ok, that's great. This means we can push this commit without immediately breaking everything in master. And the cloud team can set that var whenever they want to enable bifrost. Let me know what name would you prefer.

Reply:
Yeah, exactly! Thanks!

Reply:
Sounds good. Will update.